charts/kusion/README.md.gotmpl

```diff
@@ -40,9 +46,9 @@ Edit the [values file](values.yaml) to add your kubeconfig content
 # Replace the content below with your actual kubeconfig
 kubeconfig:
   kubeConfigs:
-    kubeconfig0: |
+    kubeconfig-0: |
       Please fill in your KubeConfig contents here.
-    kubeconfig1: |
```

Comment:
The base64 encoding command could be simplified to just 'base64 -w 0' instead of using 'cat' and 'tr -d'. This would make the command more efficient and easier to read.